bitvision/bitvision.py

```diff
@@ -15,126 +15,160 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read("config.ini")
 
-
 #########
 # HELPERS
 #########
 
+# BitVision Command Line Trading Platform
+# Aaron Lichtman and Jon Shobrook
 
-def login():
+import os
+import sys
+import urwid
+
+# Place Your Own Trade is not shown until Trading Deactivated.
+choices = ["Activate/Deactivate Automated Trading", "Deposit/Withdraw Money", "Place Your Own Trade"]
+
+
+def menu(title, choices):
+	"""
+    Make a menu with choices displayed.
     """
+	body = [urwid.Text(title), urwid.Divider()]
+	for c in choices:
+		button = urwid.Button(c)
+		urwid.connect_signal(button, 'click', item_chosen, c)
+		body.append(urwid.AttrMap(button, None, focus_map='reversed'))
+	return urwid.ListBox(urwid.SimpleFocusListWalker(body))
+
+
+def item_chosen(button, choice):
```

Comment:
I don't think this is necessary. When a menu option is chosen, a new widget will be drawn (ex: in the case of withdrawing money, a prompt will show up asking how much to withdraw) and/or something will happen in the background. All this will be handled in the `handle_input` function in the `App` class.

Reply:
I removed this block of code in favor of the footer menu bar.